dict_tools/__init__.py

```diff
@@ -7,6 +7,29 @@ def map_keys(function, dictionary):
 
 def map_values(function, dictionary):
     return {
-        key: function(value),
+        key: function(value)
+        for key, value in dictionary.items()
+    }
+
+
+def map_all(function, dictionary):
+    return {
+        function(key): function(value)
```

Comment:
This doesn't seem like a very useful function. I would expect a key mapping function and value mapping function instead.

Reply:
Yeah, it's very weird to map both keys and values with the same function if you think about it, especially since the keys and values are usually of different types